File: backend/api/alpaca.py
import websocket, json

baseUrl = 'https://paper-api.alpaca.markets'
api_key = 'PK2SHB7J8CDRIG2QJYE7'
api_secret = 'dWr71FlmFobNoWRtVVYomEPOP43yp4lP8sBbHgG9'

def on_open(ws):
    print('Opened')


def on_message(ws, message):
    print("Message: {}".format(message))

def on_close(ws):
    print('Closed')

socket = "wss://data.alpaca.markets/stream"

ws = websocket.WebSocketApp(socket, on_open=on_open, on_close=on_close, on_message=on_message)

ws.run_forever()

File: frontend/src/config/alpaca.js
export const alpaca = {
    api_token: 'PK5H79M3BFG7S451J7WD',
    secret_key: 'Dr8MM3XOnAIczPRUHcvzeW0gFBcJlP1Tn2vlOHV1',
    baseURL: 'https://data.alpaca.markets/v1',
    baseStreamURL: 'wss://data.alpaca.markets/stream'
}

export const auth_data = {
    "action": "authenticate",
    "data": {
        "key_id": alpaca.api_token,
        "secret_key": alpaca.secret_key
    }
}
